omero/developers/Python.rst

```diff
@@ -222,6 +222,37 @@ Read data
             for image in dataset.listChildren():
                 print_obj(image, 4)
 
+-  **Get Objects by their ID or attributes**
+
+   The first argument for ``conn.getObjects()`` or ``conn.getObject()`` is the object type.
+   This is not case sensitive. Supported types are
+   ``project``, ``dataset``, ``image``, ``screen``, ``plate``, ``plateacquisition``, ``acquisition``, ``well``,
+   ``roi``, ``shape``, ``experimenter``, ``experimentergroup``, ``originalfile``, ``fileset``, ``annotation``
+
+::
+
+    # Find objects by ID
+    projects = conn.getObjects("Project", [1, 2, 3])
+
+    # Single object. IDs are unique across all types of annotations
+    image = conn.getObject("Annotation", 1)
+
+    # Find an Object by attribute. E.g. 'name'
+    images = conn.getObjects("Image", attributes={"name": name})
+
+-  **Get different types of Annotations***
+
+   Supported types are: ``tagannotation``, ``longannotation``, ``booleanannotation``, ``fileannotation``,
+   ``doubleannotation``, ``termannotation``, ``timestampannotation``, ``mapannotation``
+
+::
+
+    # List All Tags
+    conn.getObjects("TagAnnotation")
```

Comment:
does that get all tags on the server ? I guess not - the restriction is coming from the "conn" - is that explained elsewhere in the doc ?